src/applications/differential/xaction/DifferentialRevisionPlanChangesTransaction.php

```diff
@@ -75,13 +75,13 @@ protected function validateAction($object, PhabricatorUser $viewer) {
       }
     }
 
-    if ($object->isChangePlanned()) {
-      throw new Exception(
-        pht(
-          'You can not request review of this revision because this '.
-          'revision is already under review and the action would have '.
-          'no effect.'));
-    }
+    // if ($object->isChangePlanned()) {
```

Comment:
Why would this be necessary? This is DifferentialRevisionPlanChangesTransaction, which almost certainly shouldn't be getting invoked when running `arc diff`?